types/hapi/definitions/server/server-method.d.ts

```diff
@@ -0,0 +1,49 @@
+import * as catbox from "catbox";
+
+/**
+ * The method function with a signature async function(...args, [flags]) where:
+ * * ...args - the method function arguments (can be any number of arguments or none).
+ * * flags - when caching is enabled, an object used to set optional method result flags:
+ * * * ttl - 0 if result is valid but cannot be cached. Defaults to cache policy.
+ * For reference [See docs](https://github.com/hapijs/hapi/blob/master/API.md#-servermethodname-method-options)
+ */
+export interface ServerMethod {
+//    (args: any[], flags: {ttl?: number}): any;
+    (...args: any[]): any; // TODO it needs review. Look the doc: the first param is a ...args and the second is "flags". Typescript supports ...args only in the last param
```

Comment:
This should probably be a type, not an interface?

Reply:
Commit b923c71. Thanks!